Update text oct24 im

iainma09 opened the pull request.

iainma09 committed:
replace swab positivity with test positivity

iainma09 committed:
typo in a seasonal coronavirus CARI title

iainma09 committed:
Updates to metadata sections

iainma09 committed:
test positivity change

lynseywaugh commented:
I'm struggling to run the app to check the changes but reviewing the branch on gitea there are a few wee changes needed:

hmpv CARI server - lines 7 & 15 - you've replaced Swab with Twab, just needs changed to Test

influenza CARI server - line 18 - need a space between test and positivity

seasonal coronavirus UI - line 36 - need a space between test and positivity

iainma09 commented:
Thanks Lynsey, I've corrected the hmpv and influenza, I'd already spotted  the seasonal  coronavirus space and corrected it in a separate commit. All good to merge.

iainma09 committed:
Lynsey's edits
